build.ps1

```diff
@@ -1,6 +1,6 @@
 $ErrorActionPreference = "Stop"
 
-dotnet test -c Release `
+dotnet test -c Release -l GitHubActions `
```

Comment:
Would be nice as you suggested for the package to auto detect, since folks (I for one) use these before pushing to remote